packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx

```diff
@@ -221,7 +221,7 @@ export const PageTwoStepAuthentication = (_: RouteComponentProps) => {
                     attrs={{ alt: true }}
                   >
                     <img
-                      className="mx-auto w-48 h-48 qr-code-border"
```

Comment:
I removed the `qr-code-border` class in favor of just adding the utility classes directly, we're not using it anywhere else.